graspy/match/faq.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import numpy as np
+import math
+import matplotlib.pyplot as plt
+from scipy.optimize import linear_sum_assignment
+from scipy.optimize import minimize_scalar
+from scipy.optimize import minimize
+from sinkhorn_knopp import sinkhorn_knopp as skp
```

Comment:
@j1c thoughts on this? Don't know if I want to add this as a dependency, other option would be to hard code it in graspy I guess?